nextcloudappstore/core/templates/app/edit.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'app/base.html' %}
+{% load i18n staticfiles humanize %}
+
+{% block head-title %}{% blocktrans with appname=object.name %}Edit app: {{ app_name }}{% endblocktrans %} - {% trans 'Apps' %} - {% endblock %}
+{% block head %}
+    <script defer="defer" src="{% static 'assets/js/ui/app-edit.js' %}"></script>
+    <meta name="language-code" content="{{ request.LANGUAGE_CODE }}">
+{% endblock %}
+
+{% block apps %}
+    <div class="header-wrap">
+        <h1>{% blocktrans with app_name=object.name %}Edit app: {{ app_name }}{% endblocktrans %}</h1>
```

Comment:
Should be **Edit app**. In general I'd avoid blocktrans since its harder to translate for translators.